spec/rubocop/cop/style/alias_spec.rb

```diff
@@ -208,5 +216,23 @@ def foo
         end
       RUBY
     end
+
+    it 'register an offense for alias_method when calling with no arguments' do
+      expect_offense(<<~RUBY)
+        alias_method
+        ^^^^^^^^^^^^ Use `alias` instead of `alias_method` at the top level.
+      RUBY
+
+      expect_no_corrections
+    end
+
+    it 'registers an offense for alias_method when calling with one argument' do
+      expect_offense(<<~RUBY)
+        alias_method :foo
+        ^^^^^^^^^^^^ Use `alias` instead of `alias_method` at the top level.
+      RUBY
+
+      expect_no_corrections
```

Comment:
Since their usage as `alias_method(new, original)` is not completed, I think they should be ignored instead of registering an offense.

Reply:
That works for me.